prisma-local/schema.prisma

```diff
@@ -267,4 +267,13 @@ model Invoice {
 
   @@unique([invoiceNumber, userId], map: "Invoice_invoiceNumber_userId_unique_constraint")
   @@unique([transactionId, userId], map: "Invoice_transactionId_userId_unique_constraint")
+}
+
+model ClientPayment {
+  id                   String                 @id @default(dbgenerated("(uuid())"))
+  paymentId            String                 @unique
+  status               String               
+  address              String
```

Comment:
Maybe is a good call to also add a foreign relation here:
call this one `addressString` and below add

```
  address      Address   @relation(fields: [addressString], references: [address])
```

...this way we can access easier the relation using prisma ORM in the TS code.

(will also have to add
```
clientPaymentIds                     ClientPaymentId[]
```
in the address table)